src/schematools/__init__.py

```diff
@@ -4,3 +4,4 @@
 MAX_TABLE_NAME_LENGTH: Final[int] = 63  # limitation of PostgreSQL
 TMP_TABLE_POSTFIX: Final[str] = "_new"
 TABLE_INDEX_POSTFIX: Final[str] = "_idx"
+DATABASE_SCHEMA_NAME_DEFAULT: Final[str] = "PUBLIC"
```

Comment:
Just a question, why schema names in all caps? Is'nt is more common practice to lowercase all identifiers for PostgreSQL?

Reply:
You are right, no reason to make it upper, lower cased it